tox.ini

```diff
@@ -0,0 +1,24 @@
+[tox]
+envlist = test-{cli,config},codestyle
+
+[testenv]
+changedir =
+    cli: tests/cli-flags
+    config: tests/config-flags
+setenv =
+    cli: NUMPY_EXPECTED=1.17.4
+    config: NUMPY_EXPECTED=1.17.5
+commands =
+    cli: tox -r -e test --pypi-filter='numpy==1.17.4'
+    cli: tox -r -e test --pypi-filter='numpy<1.17.5'
+    cli: tox -r -e test --pypi-filter='file://constraints.txt'
```

Comment:
I think you could add an absolute path test using `{toxinidir}`

Reply:
Done!